src/mapd-client-v2.js

```diff
@@ -1,4 +1,4 @@
-/* global MapDClient*/
+const MapDClient = (typeof window !== "undefined" && window.MapDClient) || require("../build/thrift/node/mapd.thrift.js").Client // eslint-disable-line global-require
```

Comment:
Two concerns with this file: 
1. It's only used by the browser version of connector, which makes it quite likely that someone will add something that works in node and forget to change it here.
2. Also the covered methods have to be manually enumerated, which is brittle.